File: fizzBuzz.java
public class fizzBuzz{
  public static void main(String[] args){

    String ans = " ";

    for(int x = 0; x++ < 100;){
      System.out.print("> ");
      if(x % 3 == 0 && x % 5 == 0){
        System.out.println("Fizz Buzz");
      } else if(x % 3 == 0){
        System.out.println("Fizz");
      } else if(x % 5 == 0){
        System.out.println("Buzz");
      } else{
        System.out.println(x);
      }
    }
  }
}

File: fizzBuzz.cpp
#include <iostream.h>
#include <string.h>

using namespace std;

int main(){

	int n;
	cout << "What number would you like to play FizzBuzz to?";
	cin >> n;
	string result;
	
	for(int x = 1; x <= n; x++){
		result = "";
		if(x % 3 == 0){
			result += "Fizz";
		}
		if(x % 5 == 0){
			result += "Buzz";
		}
		if(result == "" ){
			result = (to_string(x));
		}
		cout << (result + "\n");
	}
	return 0;
}